packages/discovery/tsconfig.json

```diff
@@ -6,5 +6,5 @@
     "tsBuildInfoFile": "dist/.tsbuildinfo"
   },
   "include": ["src"],
-  "exclude": ["src/**/*.spec.ts", "src/test_utils"]
```

Comment:
why is it removed?

Reply:
`src/test_utils` isn't actually any directory in this package. seems to have come in during a copy. `spec.ts` files are still excluded